lib/tenji_maker.rb

```diff
@@ -1,43 +1,27 @@
+require_relative './tenji_maker/base'
+require_relative './tenji_maker/upper'
+require_relative './tenji_maker/middle'
+require_relative './tenji_maker/lower'
+
 class TenjiMaker
+  TENJI_POINTS = { all: "oo", left: "o-", right: "-o", none: "--"}.freeze
```

Comment:
```suggestion
  TENJI_POINTS = { all: "oo", left: "o-", right: "-o", none: "--" }.freeze
```

のように最後にスペースを空けた方がいいですね。